start

```diff
@@ -491,15 +497,69 @@ function stop_postgres() {
   while kill -0 "$CURRENT_POSTGRES_PID" &> /dev/null; do
     sleep 0.5
   done
-  echo "postgres: down"
+  print_output "postgres: down"
 }
 
 pushd () {
-    command pushd "$@" > /dev/null
+  command pushd "$@" > /dev/null
 }
 
 popd () {
-    command popd "$@" > /dev/null
+  command popd "$@" > /dev/null
+}
+
+function soroban_rpc_status () {
+  horizon_status
+  print_colored_screen_output "waiting for soroban rpc to report ready state..." "\033[34m" 
+  COUNTER=1
+  while ! $(curl --silent --location --request POST 'http://localhost:8000/soroban/rpc' \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+        "jsonrpc": "2.0",
+        "id": 10235,
+        "method": "getHealth"
+        
+    }' | jq --exit-status '.result.status == "healthy"' 2>/dev/null | grep -o true || echo false ); 
+  do
+    if [ $COUNTER -gt 12 ]; then 
```

Comment:
Fyi I've noticed there's a lot of stray trailing whitespace:
<img width="413" alt="Screenshot 2023-01-20 at 3 28 47 PM" src="https://user-images.githubusercontent.com/351529/213822968-48a54d53-396b-4987-a102-c62ba1cae455.png">

Reply:
I used ide to reformat script to 2 space tabs.